src/main/java/net/imglib2/RealRandomAccessibleRealInterval.java

```diff
@@ -43,6 +43,10 @@
  * A function over an n-dimensional real interval that can create a random
  * access {@link Sampler}.
  * </p>
+ * <p>
```

Comment:
I do not think this is true.  Typically, a RealRandomAccessibleInterval is expected to be defined everywhere in the interval.  Where is this not true (while not being an errror)?